docs/data-access/data-access.md

```diff
@@ -19,9 +19,9 @@ As a prerequisite to applying for DACO access, please ensure that you:
 
 ## How to Apply for Access through DACO
 
-The instructions for applying for DACO can be found here on the [ICGC DACO website](https://icgc.org/daco), as well as a selection of [help guide videos](https://icgc.org/daco/help-guide-section).
+The instructions for applying for DACO can be found here on the [ICGC DACO website](https://daco.icgc.org/), as well as a selection of [help guide videos](https://daco.icgc.org/help-guides/daco-help-guides/).
 
-1. Register a DACO account at [www.icgc.org/user/register](https://icgc.org/user/register). If you already have an account, you can log in from here [www.icgc.org/user/login](https://icgc.org/user/login).
+1. Register a DACO account at https://daco.icgc.org/user/#/register. If you already have an account, you can log in from https://daco.icgc.org/user/#/login. 
 1. After registering, you will receive an email instructing you to set your Central User Directory password. Click the link from the email to set up your account password.
 1. Once you save your password, you will be taken to your ICGC Dashboard. From your Dashboard, click the link to start a new access application form from the box entitled “Your DACO Access Forms”.
```

Comment:
Once you save your password, you will be taken to your ICGC Dashboard. From your Dashboard, click the "Start New Application" button the box entitled “Manage Your Submissions”.

Reply:
Rosi I don't know if the rest of the instructions are correct - I think the application has changed where you can print or sign it digitally?

Reply:
I know, but this should be good enough to get them to the right place now?

Reply:
we could remove the rest of the instructions then as they could be confusing

Reply:
lets leave it for now just until we know a bit more.  I can update again next week.  i dont think this is the most urgent.